data/defscript/popups.kvs

```diff
@@ -742,7 +742,7 @@ defpopup(windowpopup)
 
 	separator
 
-	item($tr("&Option Controllers","defscript"),$icon("options"))
+	item($tr("&Global Option Controllers","defscript"),$icon("options"))
```

Comment:
I think the `&` makes more sense to be on the `O` instead of the `G` for the options menu.